src/main/java/me/karakelley/http/ResponseFormatter.java

```diff
@@ -1,31 +0,0 @@
-package me.karakelley.http;
-
-import java.util.Map;
-import java.util.stream.Collectors;
-
-public class ResponseFormatter {
```

Comment:
Why did you decide to move all of this into `Response`?

Reply:
When I was changing the functionality of `ResponseFormatter` it made sense that this belonged in `Response`.